<!--
import data_analytics.github as github
print(github.create_jupyter_notebook_header("marcocrowe", "learn-mathematical-methods", "exam-2024-mathematical-methods-2-math06100-solution.ipynb", "master"))
-->
<table style="margin: auto;"><tr><td><a href="https://mybinder.org/v2/gh/marcocrowe/learn-mathematical-methods/master?filepath=exam-2024-mathematical-methods-2-math06100-solution.ipynb" target="_parent"><img src="https://mybinder.org/badge_logo.svg" alt="Open In Binder"/></a></td><td>online editors</td><td><a href="https://colab.research.google.com/github/marcocrowe/learn-mathematical-methods/blob/master/exam-2024-mathematical-methods-2-math06100-solution.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a></td></tr></table>


# Mathematical Methods 2 - Exam Paper - 2023 - Summer

## Setup

In [4]:
from sympy import diff, exp, cos, latex, ln, sin, solve, symbols, integrate
from IPython.display import display, Markdown

def calculate_and_display_derivative(title:str , x, x_equation, y_name:str="y"):
    dx_dy = diff(x_equation, x)
    # Display the derivative in LaTeX format
    display(Markdown(f"### {title}"))
    display(Markdown("$\Large\n" + y_name + " = " + latex(x_equation) + r" \implies \\ \frac{d" + y_name + "}{d" + x.name + "} = " + latex(dx_dy) + "$"))

## Instructions


**Time Allowed**: 2 hours

**Instructions**:  Answer 4 questions. All questions carry equal marks

The use of programmable or text storing calculators is expressly forbidden.

Please note that where a candidate answers more than the required number of questions, the examiner will mark all questions attempted and then select the highest scoring ones.

***Requirements for this paper:***

1. ***Log Tables***
2. ***Calculators***


## Question 1 (Total Marks 25)

### Question 1(a) [12 Marks]

Solve the triangle shown below:

![Figure 1](images/exam-2024-mathematical-methods-2-math06100-paper/figure-1.png)

### Question 1(b) [3 Marks]

When the angle of elevation of the sun is 10°, an upright flagpole casts a shadow of length of 20m. Calculate the height of the pole.

### Question 1(c) [4 Marks]



Given $2.3\ln{\dfrac{1.3}{x^2}}=9.1$, Solve for x:

#### Question 1(c) Answer

<details>
<summary>A Note on ln</summary>
What is the difference between ln and log?

$\ln$ is the natural logarithm i.e. $$\ln{x} = \log_{e}{x}$$

So for any of the log rules, e.g the quotient rule:  
$$\log_{c}{\dfrac{a}{b}} = \log_{c}{a} - \log_{c}{b}$$  
for natural logarithms, substitute $c=e$ to get the natural logarithm rule for quotients:  
$$\log_e{\dfrac{a}{b}} = \log_e{a} - \log_e{b}$$  
and $\log_e{x} = \ln{x}$ so the rule becomes:  
$$\ln{\dfrac{a}{b}} = \ln{a} - \ln{b}$$

---
</details>

$2.3\ln{\dfrac{1.3}{x^2}}=9.1$

The log quotient rule $\ln{\dfrac{a}{b}} = \ln{a} - \ln{b}$ applies to $\ln{\dfrac{1.3}{x^2}}$  
$\therefore \ln({\dfrac{1.3}{x^2}}) = \ln({1.3}) - \ln({x^2})$

$2.3(\ln({1.3}) - \ln({x^2}))=9.1$  
$2.3\ln({1.3}) - 2.3\ln({x^2})=9.1$

The power rule for logarithms states that $\log_{b}{x^n} = n\log_{b}{x}$
$\therefore \ln({x^2}) = 2\ln({x})$

$2.3\ln({1.3}) - 2.3(2\ln({x}))=9.1$  
$2.3\ln({1.3}) - 4.6\ln({x})=9.1$  
$2.3\ln({1.3}) - 9.1=4.6\ln({x})$  
$\dfrac{2.3\ln({1.3}) - 9.1}{4.6}=\ln({x})$  
$\ln({x}) = \dfrac{2.3\ln({1.3}) - 9.1}{4.6}$

Inverse Property of Logarithms and Exponentials: If $ln(a)=b$, then $a=e^b$.  
This means that for any real number $k$, $ln(x)=k$ is equivalent to $x=e^k$.

Let $k=\dfrac{2.3\ln({1.3}) - 9.1}{4.6}$
$ln(1.3) \approx 0.2624$

$k \approx \frac{2.3 \times 0.2624 - 9.1}{4.6} \approx \frac{0.6048 - 9.1}{4.6} \approx \frac{-8.4952}{4.6} \approx -1.8470$

$x = e^{\dfrac{2.3\ln({1.3}) - 9.1}{4.6}}$

$\therefore ln(x)\approx k \approx -1.8470$
$\therefore x=e^{-1.8470} \approx 0.157$

$\boxed{x \approx 0.157}$

In [5]:
# Define the variable
x = symbols('x')

# Define the equation
equation = 2.3 * ln(1.3 / x**2) - 9.1

# Isolate the logarithm term
isolated_log = solve(equation, x)

isolated_log


[-0.157697168944812, 0.157697168944812]

The SymPy library provides two solutions to the equation

However, since the logarithm of a negative number is not defined in the real number system, the valid solution is:

$ x \approx 0.1577 $

### Question 1(d) [6 Marks]

For the function shown below, determine the second order differential $f(x)$.

$f(x) =-6x+\frac{1}{\sqrt[3]{x}} - \frac{1}{4x^3}$

#### Question 1(d) Answer

The first order differential of $f(x)$ is: $f'(x)$  
The second order differential of $f(x)$ is: $f''(x)$  i.e. the derivative of $f'(x) = f''(x)$

$f(x) =-6x+\frac{1}{\sqrt[3]{x}} - \frac{1}{4x^3}$  


The sum and difference rules to differentiate $f(x)$ are:

$$f(x) = g(x) + h(x) \implies f'(x) = g'(x) + h'(x)$$

$$f(x)= g(x) - h(x) \implies f'(x) = g'(x) - h'(x)$$

For each term in $f(x)$, the derivative is:

1. $g(x) = -6x$  
2. $h(x) = \frac{1}{\sqrt[3]{x}}$  
3. $i(x) = \frac{1}{4x^3}$

For (1) $g(x)$ we apply the constant multiple rule:

$$g(x) = c \cdot x \implies g'(x) = c$$

$$\dfrac{d}{dx}(-6x) = -6\dfrac{d}{dx}(x) = -6$$

For 2. $h(x)$ we apply the power rule:

$$h(x) = a \cdot x^n \implies h'(x) = a \cdot n \cdot x^{n-1}$$  
$h(x) = \frac{1}{\sqrt[3]{x}} = x^{-\frac{1}{3}}$, $a=1$, $n=-\frac{1}{3}$  
$h'(x) = -\frac{1}{3} \cdot x^{(-\frac{1}{3}-1)} = -\frac{1}{3} \cdot x^{-\frac{4}{3}} = -\frac{1}{3} \cdot \frac{1}{x^{\frac{4}{3}}} = -\frac{1}{3x^{\frac{4}{3}}}$  

For 3. $i(x)$ we apply the power rule:

$$i(x) = a \cdot x^n \implies i'(x) = a \cdot n \cdot x^{n-1}$$  

$i(x) = \frac{1}{4x^3}$ = $\frac{1}{4} \cdot x^{-3}$, $a=\frac{1}{4}$, $n=-3$  
$i'(x) = -3 \cdot \frac{1}{4} \cdot x^{(-3-1)} = -\frac{3}{4} \cdot x^{-4} = -\frac{3}{4x^4}$  


Therefore, the first order differential of $f(x)$ is:

$f'(x) = g(x) + h(x) - i(x)$  

$${f'(x) = -6 - \frac{1}{3x^{\frac{4}{3}}} - \frac{3}{4x^4}}$$

The second order differential of $f(x)$ is:

$f''(x) = \frac{d}{dx}(-6 - \frac{1}{3x^{\frac{4}{3}}} - \frac{3}{4x^4})$

The sum and difference rules to differentiate $f'(x)$ are:

$$f'(x) = g'(x) + h'(x) \implies f''(x) = g''(x) + h''(x)$$  
$$f'(x) = g'(x) - h'(x) \implies f''(x) = g''(x) - h''(x)$$  

$\therefore f''(x) = -6 - \frac{1}{3x^{\frac{4}{3}}} - \frac{3}{4x^4}$

Label the terms in the second order differential of $f(x)$ as $g'(x)$, $h'(x)$, $i'(x)$:

1. $g'(x) = -6$  
2. $h'(x) = -\frac{1}{3x^{\frac{4}{3}}} = -\frac{1}{3} \cdot \frac{1}{x^{\frac{4}{3}}} = -\frac{1}{3} \cdot x^{-\frac{4}{3}}$  
3. $i'(x) = -\frac{3}{4x^4} = -\frac{3}{4} \cdot x^{-4}$

Differentiating $g'(x)$, $h'(x)$, $i'(x)$:

1. Applying the constant multiple rule to $g'(x) = -6$:  
   $\therefore g'(x) = -6$  
2. Applying the power rule to $h'(x), a=-\frac{1}{3}$, $n=-\frac{4}{3}$  
   $\therefore h''(x) = -\frac{4}{3} \cdot -\frac{1}{3} \cdot x^{(-\frac{4}{3}-1)} = \frac{4}{9} \cdot x^{-\frac{7}{3}} = \frac{4}{9x^{\frac{7}{3}}}$  
3. Applying the power rule to $i'(x), a=-\frac{3}{4}$, $n=-4$  
   $\therefore i''(x) = -4 \cdot -\frac{3}{4} \cdot x^{(-4-1)} = 3 \cdot x^{-5} = \frac{3}{x^5}$

Therefore, the second order differential of $f(x)$ is:

$f''(x) = 0 + \frac{4}{9x^{\frac{7}{3}}} + \frac{3}{x^5}$

$$\boxed{f''(x) = \frac{4}{9x^{\frac{7}{3}}} + \frac{3}{x^5}}$$











### Question 2(b) [8 Marks]


Differentiate the following with respect to the variable:

1. $y = \dfrac{5}{3e^{4x}}$
2. $y = \frac{4}{7}e^{-\frac{2}{4}x}$
3. $y = \frac{2}{3}\ln{3\pi x}$
4. $y = 3\sin{2\theta} - 2\cos{4\theta}$

#### Question 2(b).3 Answer

Differentiate $y = \frac{2}{3}\ln{3\pi x}$  

Log Rule: $\log(ab) = \log(a) + \log(b)$

$\ln{3\pi x} = \ln{3} + \ln{\pi} + \ln{x}$

So y = $\frac{2}{3}(\ln{3} + \ln{\pi} + \ln{x})$

The sum rule for differentiation applies $\dfrac{d}{dx}(f(x) + g(x)) = f'(x) + g'(x)$

Label the terms in $y$ as $g(x)$, $h(x)$, $i(x)$:

Differentiating $g(x)$, $h(x)$, $i(x)$:

1. $g(x) = \frac{2}{3}\ln{3}$
2. $h(x) = \frac{2}{3}\ln{\pi}$
3. $i(x) = \frac{2}{3}\ln{x}$

The derivative of log of a constant is zero, so the derivative of $g(x)$, $h(x)$, $i(x)$ is:

1. $\dfrac{d}{dx}(\frac{2}{3}\ln{3}) = 0$  
2. $\dfrac{d}{dx}(\frac{2}{3}\ln{\pi}) = 0$  
3. $\dfrac{d}{dx}(\frac{2}{3}\ln{x}) = \frac{2}{3} \cdot \frac{1}{x} = \frac{2}{3x}$:


Sum the derivatives of $g(x)$, $h(x)$, $i(x)$:

$\dfrac{d}{dx}(0 + 0 + \frac{2}{3x}) = 0 + 0 + \frac{2}{3x}$

$$\boxed{\dfrac{d}{dx}(\frac{2}{3}\ln{3\pi x}) = \frac{2}{3x}}$$

Note: if you Have $k\ln({c \cdot x}), \text{ where k and c are constants, then the derivative is } k\cdot \dfrac{1}{x}$

#### Question 2(b).4 Answer

$y = 3\sin{2\theta} - 2\cos{4\theta}$

The difference rule applies to the differentiation of $y$:

$$\dfrac{d}{dx}(f(x) - g(x)) = f'(x) - g'(x)$$

Label the terms in $y$ as $g(x)$, $h(x)$:

1. $g(x) = 3\sin{2\theta}$
2. $h(x) = 2\cos{4\theta}$

Differentiating $g(x)$, $h(x)$:

1. $g(x) = 3\sin{2\theta}$  The rule for differentiating $\sin{ax}$ is $a\cos{ax}$  
   $\dfrac{d}{dx}(3\sin{2\theta}) = 3 \cdot 2 \cos{2\theta} = 6\cos{2\theta}$
2. $h(x) = 2\cos{4\theta}$  The rule for differentiating $\cos{ax}$ is $-a\sin{ax}$  
   $\dfrac{d}{dx}(2\cos{4\theta}) = 2 \cdot -4 \sin{4\theta} = -8\sin{4\theta}$

Subtract the derivatives of $g(x)$, $h(x)$:

$\dfrac{d}{dx}(6\cos{2\theta} - (- 8\sin{4\theta})) = 6\cos{2\theta} + 8\sin{4\theta}$

$$\boxed{\dfrac{d}{dx}(3\sin{2\theta} - 2\cos{4\theta}) = 6\cos{2\theta} + 8\sin{4\theta}}$$

In [3]:
#$y = 3\sin{2\theta} - 2\cos{4\theta}$

theta = symbols(r'\theta')
y = 3 *sin(2*theta) - 2*cos(4*theta)

calculate_and_display_derivative("Question 2.b.4", theta, y, 'y')

### Question 2.b.4

$\Large
y = 3 \sin{\left(2 \theta \right)} - 2 \cos{\left(4 \theta \right)} \implies \\ \frac{dy}{d\theta} = 8 \sin{\left(4 \theta \right)} + 6 \cos{\left(2 \theta \right)}$


### Question 2(c) [6 Marks]


Graph the function $y = \cos{2\theta}$ for $0° \geq \theta \leq 180°$ in intervals of 20°.


### Question 2(d) [4 Marks]


Solve the triangle, given one side of a triangle and two angles are:

$$a = 2, A = 30°, B = 40°$$


## Question 3 (Total Marks 25)

### Question 3(a) [15 Marks]

The number of cells N in a bacterial population in time t hours from the commencement of growth is given by $N = N_0e^{kt}$. Assuming the starting size $N_0 = 110$, and $k=1.7$.

1. Plot the graph representing N bacterial cells for 0 ≤ t ≤ 2 hours, in half-hour steps. **(8 marks)**
2. Determine the size of the population after 10.25 hours using both the formula and graph methods. **(4 marks)**
3. Determine the time for the population to increase 10 times the initial value. **(2 marks)**

### Question 3(b) [4 Marks]

Convert:

1. $\dfrac{5 \pi}{6}$ to degrees. **(2 marks)**
2. $180°$ to radians. **(2 marks)**

### Question 3(c) [6 Marks]

Use the chain rule to differentiate the following:

$$y = \sqrt{2x-1}$$

## Question 4 (Total Marks 25)

### Question 4(a) [4 Marks]

Use the product rule to differentiate the following:

$$y = x^2 \cos{2x}$$

### Question 4(b) [4 Marks]

Use the quotient rule to differentiate the following:
$$f(x) = \dfrac{x^2+1}{e^{2x}}$$

### Question 4(c) [8 Marks]

A man leaves a point walking at 5 km/h in a direction E30°N . A cyclist leaves the same point at the same time in a direction E40°S travelling at a constant velocity. Find the average velocity of the cyclist if the walker and cyclist are 60 km apart after 3.5 hours.

### Question 4(d) [9 Marks]

The isotope Technitium-99m has a decay constant of $3.2 \times 10^{-5}/sec$, you measure a current weight of 4 grams of the isotope. Given the following formula:

$$m = m_0e^{-\lambda t}$$

Determine:

1. The mass of the isotope 24 hours previous to the initial measurement. **(5 marks)**
2. The half-life in hours. **(4 marks)**

## Question 5 (Total Marks 25)

### Question 5(a) [12 Marks]

An objects position $S$ in relation to time $t$ can be expressed by the following equation:

$$S = 2t^3 - 3t^2 + t + 6$$

1. Derive the equations for the object's velocity, and acceleration. **(2 marks)**
2. On a single graph plot the objects position, velocity and acceleration from time 0 to 2.5 in intervals of 0.5 seconds. **(8 marks)**
3. Using ***your graph*** determine objects position, velocity and acceleration after 0.75 seconds. **(2 marks)**



### Question 5(b) [8 Marks]


1. $\int 4 + \frac{3}{7}x - 6x^2\ dx$ **(4 marks)**
2. $\frac{3}{x^2} + 3\sqrt{x}+3x^2 - 5\ dx$ **(4 marks)**


To solve the the integrals, use the following rules:

1. $\int a\ dx = ax + C$  
2. $\int x^n\ dx = \frac{x^{n+1}}{n+1} + C$
3. $\int \frac{1}{x}\ dx = \ln{|x|} + C$
4. $\int e^x\ dx = e^x + C$
5. $\int \sin{x}\ dx = -\cos{x} + C$
6. $\int \cos{x}\ dx = \sin{x} + C$

\int 4 + \frac{3}{7}x - 6x^2\ dx$

Integrate each term separately:

1. $\int 4\ dx = 4x + C_1$
2. For $\int \frac{3}{7}x\ dx$, apply the power rule: (Note x = $x^1$)  
   $\int x^n\ dx = \frac{x^{n+1}}{n+1} + C$  
   $\int \frac{3}{7}x\ dx = \frac{3}{7} \cdot \frac{x^{1+1}}{1+1} + C = \frac{3}{7} \cdot \frac{x^2}{2} + C = \frac{3}{14}x^2 + C_2$  
3. For $\int -6x^2\ dx$, apply the power rule:  
    $\int -6x^2\ dx = -6 \cdot \frac{x^{2+1}}{2+1} + C = -6 \cdot \frac{x^3}{3} + C = -2x^3 + C_3$

Combine the integrals:

Let $C = C_1 + C_2 + C_3$

$$\int 4 + \frac{3}{7}x - 6x^2\ dx = 4x + \frac{3}{14}x^2 - 2x^3 + C$$

$$\boxed{4x + \frac{3}{14}x^2 - 2x^3 + C}$$

In [11]:
x= symbols('x')
#\int 4 + \frac{3}{7}x - 6x^2\ dx
equation = 4 + 3/7*x - 6*x**2

integral= integrate(equation, x)
# Define the constant of integration
C = symbols('C')

# Include the constant in the final result
integral_with_constant = integral + C

# Display the result
integral_with_constant


C - 2.0*x**3 + 0.214285714285714*x**2 + 4.0*x

#### Question 5(b).2 Answer

To integrate the function $\frac{3}{x^2} + 3\sqrt{x} + 3x^2 - 5$ with respect to $x$, we will break it down and integrate each term separately.

Given:
$$\int \left( \frac{3}{x^2} + 3\sqrt{x} + 3x^2 - 5 \right) \, dx$$

We can rewrite the integrand using exponents for clarity:
$$\frac{3}{x^2} = 3x^{-2}$$
$$3\sqrt{x} = 3x^{1/2}$$

Thus, the integral becomes:
$$\int \left( 3x^{-2} + 3x^{1/2} + 3x^2 - 5 \right) \, dx$$

Now, integrate each term separately:

1. **Integral of $3x^{-2}$:**
   $$\int 3x^{-2} \, dx = 3 \int x^{-2} \, dx = 3 \left( \frac{x^{-1}}{-1} \right) = -3x^{-1} = -\frac{3}{x}$$

2. **Integral of $3x^{1/2}$:**
   $$\int 3x^{1/2} \, dx = 3 \int x^{1/2} \, dx = 3 \left( \frac{x^{3/2}}{\frac{3}{2}} \right) = 3 \cdot \frac{2}{3} x^{3/2} = 2x^{3/2}$$

3. **Integral of $3x^2$:**
   $$\int 3x^2 \, dx = 3 \int x^2 \, dx = 3 \left( \frac{x^3}{3} \right) = x^3$$

4. **Integral of $-5$:**
   $$\int -5 \, dx = -5x$$

### Combine the Results:

Summing these results and including the constant of integration $C$:
$$\int \left( \frac{3}{x^2} + 3\sqrt{x} + 3x^2 - 5 \right) \, dx = -\frac{3}{x} + 2x^{3/2} + x^3 - 5x + C$$

Thus, the indefinite integral is:

$$\boxed{-\frac{3}{x} + 2x^{3/2} + x^3 - 5x + C}$$

In [10]:
# $\frac{3}{x^2} + 3\sqrt{x}+3x^2 - 5\
x= symbols('x')
equation = 3/x**2 + 3*x**0.5 + 3*x**2 - 5

integral= integrate(equation, x)
# Define the constant of integration
C = symbols('C')

# Include the constant in the final result
integral_with_constant = integral + C

# Display the result
integral_with_constant


C + x**3 - 5*x + 2.0*x**1.5 - 3/x


### Question 5(c) [5 Marks]



The velocity *v* of a body *t* seconds after a certain instant is:

$$(2.5t^2 + 6t)ms^{-1}$$

Find by integration how far it moves in the interval from t = 0 to t =10 s.